.github/workflows/compatibility.yml

```diff
@@ -0,0 +1,124 @@
+name: Cocmpatibility - WC, WP and PHP
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  woocommerce-compatibility:
+    name:    WC compatibility
+    runs-on: ubuntu-16.04
+    strategy:
+      fail-fast:    false
+      max-parallel: 10
+      matrix:
+        woocommerce: ['4.0.0', '4.3.0', '4.4.0', '4.5.0', '4.6.0']
+        wordpress:   ['latest']
+        php:         ['7.4']
+    steps:
+      # clone the repository
+      - uses: actions/checkout@v2
+      # enable dependencies caching
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/composer/
+          key:  ${{ runner.os }}-composer-${{ hashFiles('composer.lock') }}
+      # setup PHP, but without debug extensions for reasonable performance
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools:       composer
+          coverage:    none
+      # install dependencies and run tests
+      - run: composer self-update 2.0.6 && composer install --no-progress
+      - run: sudo systemctl start mysql.service
+      - run: WCPAY_DIR="$GITHUB_WORKSPACE" bash ./bin/install-wp-tests.sh woocommerce_test root root localhost ${{ matrix.wordpress }} ${{ matrix.woocommerce }} false
+      - run: bash ./bin/phpunit.sh
```

Comment:
The syntax seem to be slightly incorrect. Found the [thread](https://stackoverflow.com/questions/59529042/difference-between-run-and-multiple-runs-in-github-actions) about it.

Reply:
Will look into the thread, but it works.

Reply:
Checked, my approach definitely has drawbacks. Since we are going to replace those with a single bash-script, we'll be on the safe side. Thank you for checking and sharing the thread!